### ISSUE TITLE
[feat]: more plugins

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/cln-startos#67]
- probing
- noise
- commando (control remote CLN nodes over the lightning protocol)
- sparko

**Comment by kn0wmad** [comment]
[Eye of Satoshi](https://github.com/talaia-labs/rust-teos) pl0x

**Comment by chrisguida** [comment]
I'll split them out once I start working on them. For now we can put them all here. TEoS has its own issue so let's just leave that there

**Comment by BitcoinMechanic** [comment]
`circular` please :)

**Comment by kn0wmad** [comment]
Adding a +1 for Commando as it will be required for integration with the Alby app.  No rush as some sort of clearnet connection is also required

**Comment by chrisguida** [comment]
Alby and clams apparently use commando

**Comment by chrisguida** [comment]
Commando is active by default since CLN 0.12. Alby is tested and works great

**Comment by chrisguida** [comment]
This one looks epic https://github.com/joelklabo/nostrify

**Comment by chrisguida** [comment]
WhatSat is for lnd, crossing that off the list... https://github.com/joostjager/whatsat

**Comment by chrisguida** [comment]
`coffee` (and probably `reckless`, come to think of it) blocked by https://github.com/coffee-tools/coffee/issues/66

**Comment by chrisguida** [comment]
coffee wip here https://github.com/Start9Labs/c-lightning-wrapper/pull/81

**Comment by kn0wmad** [comment]
These probably ought to be broken out into individual feature requests.  For example, watchtowers are now an included feature, and Runes are used instead of commando and sparko.
